packages/config-loader/src/sources/ConfigSources.ts

```diff
@@ -161,7 +161,7 @@ export class ConfigSources {
       }
       return FileConfigSource.create({
         watch: options.watch,
-        path: arg.target,
+        path: resolvePath(rootDir, arg.target),
```

Comment:
Current behavior here is to resolve relative to cwd, thinking it's best to stick to that. So it'd be just `resolvePath(arg.target)`

Reply:
makes sense!